app/services/embeddable_content/video_links/node_processor.rb

```diff
@@ -12,7 +12,7 @@ def video_player
         end
       end
 
-      PLAYER_PRESENTATION_TARGETS      = %i[cc cms exported qti web].freeze
+      PLAYER_PRESENTATION_TARGETS      = %i[cc schoology cms exported qti web].freeze
```

Comment:
ditto this and the other targets (line 16, 25); they should one day go on the ExportTarget model in all likelhood...